editor/components/warning/style.scss

```diff
@@ -1,4 +1,4 @@
-.editor-warning {
+.edit-post-visual-editor .editor-warning {
```

Comment:
Why was this change made? It regressed styling on the application-wide error.

Reply:
See also #4781

Reply:
Also, should avoid targets that are outside the scope of the component.

Reply:
Yeah, probably worth moving the post-editor-specific styles to a component in that directory.